tests/backbeat/tests/api/metrics.js

```diff
@@ -83,8 +128,8 @@ describe.skip('Backbeat replication metrics', function dF() {
                 }),
             next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(100),
                 next),
-            next => scalityUtils.compareObjectsAWS(srcBucket, destBucket, key,
-                undefined, next),
+            next => scalityUtils.compareObjectsAWS(srcBucket, awsDestBucket,
+                key, undefined, next),
             next => makeDelayedGETRequest('/_/backbeat/api/metrics/crr/all',
```

Comment:
Let's use the `doWhilst` here to wait for the expected response. I'm comfortable saying the test failed if the metrics aren't there by the time the `REPLICATION_TIMEOUT` is exceeded.